### DISCUSSION TITLE
enterprise has been suspended / no notice / no reaction from support

### DISCUSSION THREAD

**Discussion by machines-like-christopher** [community/community#141081] [Enterprise] [answered]
### Select Topic Area

Question

### Body

We are facing a severe, company-wide emergency that has completely halted our operations. Our entire team is currently unable to deploy or push any code, which is impacting a whole team of developers and leaving us unable to respond to urgent client requests. This is a crisis-level situation.

We are seeing a red banner stating, "organization has been flagged,", "enterprise has been suspended"  but we have not received any prior notification or instructions regarding necessary actions on our part.

We are waiting now since 3 days for a reply from the support team. There is nothing more then radio silence. 
We would appreciate any hint to get this escalated.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @machines-like-christopher, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. I've checked in on this ticket and you should be getting a response shortly. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
